programs/survival/portalorient.sc

```diff
@@ -0,0 +1,206 @@
+__config() ->
+(
+	{
+		['scope','player'],
+		['stay_loaded',true],
+		
+		['commands',
+			{
+				['','__guide'],
+				['<mode>','__change']
+			}
+		],
+
+		['arguments',
+			{

```

Comment:
Proper map syntax would be
```
{
    'scope' -> 'player',
    'stay_loaded' ->true,
    'commands' -> {
        '' -> '__guide',
       '<mode>' -> '__change',
    },
    'arguments' -> { ...

```
and so on

Reply:
I've stopped using `m()` and `l()` because they're "older", but the docs call my method here "equivalent" to yours and I prefer to do it this way.